test/deriver/ScopedKeys.js

```diff
@@ -70,6 +70,21 @@ describe('ScopedKeys', function () {
       });
   });
 
+  it('should correctly derive Thunderbird sync key to known test vectors', () => {
```

Comment:
These are the same values as the test for the `oldsync` key. Does it matter?

Reply:
I don't believe it should.